modules/operations/books_ops.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../book'
+
+module BookOperations
+  def list_books
+    if @books.empty?
+      puts 'No book found. Please insert one.'
+    else
+      @books.each_with_index do |book, index|
+        puts "#{index}) Title: \"#{book.title}\", Author: #{book.author} "
```

Comment:
- [ ] This book displays some data that should not be printed to the user. This does not happen to the book alone, but to the rentals also. The person creation works fine. I would advice you to fix this bug for a better user experience.

![image](https://user-images.githubusercontent.com/86200356/211374377-0cfa75f5-c05b-45e7-91e7-16624dccffd2.png)

Reply:
@Olamarx Thanks for the feedback.

Reply:
**OPTIONAL** - The issue raised by the previous reviewer still persists, I think you forgot to push the changes made. Please ensure you push the changes to fix this issue 🙏